HighwayPatrol.h

```diff
@@ -1,11 +1,14 @@
 #pragma once 
 
 #include "Vehicle.h"
+#include "Highway.h"
 
 struct HighwayPatrol : Vehicle 
 {
     HighwayPatrol();
+    ~HighwayPatrol();
     void scanHighway(Highway* h);
+    std::string vehicleType;
```

Comment:
This doesn't need to be a member variable.
It only needs to exist in the block where it is used: inside `if( willArrest )`
Always try to limit the scope/lifetime of variables to be as short as possible
- [ ] revise